x-pack/solutions/security/packages/kbn-cloud-security-posture/public/src/utils/helpers.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Truthy } from 'lodash';
+
+export const truthy = <T>(value: T): value is Truthy<T> => !!value;
```

Comment:
I noticed that we were importing only the `Truthy` type from Lodash without actually using any Lodash functions. I think this is a bit of an odd pattern because Lodash is primarily a utility library, so importing only the type instead of simply using the `truthy` function directly from lodash doesn't seem to make much sense.

If the case is that we had to customize the lodash's `truthy` function because it doesn't work for us out of the box, I think it's better to create our own `Truthy` type as well (based on lodash):

```suggestion
type Truthy<T> = T extends null | undefined | false | "" | 0 | 0n ? never : T;

export const truthy = <T>(value: T): value is Truthy<T> => !!value;
```